R/geom-.r

```diff
@@ -145,24 +145,22 @@ Geom <- gganimintproto("Geom",
   },
 
   pre_process = function(g, g.data, ranges){
-    return(list(g = g, g.data = g.data))
+    list(g = g, g.data = g.data)
```

Comment:
style: only use return for early exit

Reply:
Should we add coding styling somewhere? I think it may help new contributors.